docs/sphinx/user_guide/feature/loop_basic.rst

```diff
@@ -68,35 +62,64 @@ may be written in a RAJA form as::
     c[i] = a[i] + b[i];
   });
 
-``RAJA::forall`` templates take a template argument for the 
-execution policy and two arguments: an object describing the loop iteration 
-space, such as a RAJA segment or index set, and a lambda expression defining 
-the loop body. Applying different loop execution policies enables the loop to 
-run in different ways; e.g., using different programming model back-ends. 
-Different iteration space objects, enable the loop iterates to be reordered, 
-run in different threads, etc. 
+A ``RAJA::forall`` template is templated on an execution policy type and takes
```

Comment:
``template is templated" sounds a bit repetitive could we say "A ``RAJA::forall`` method is templated ..."?

Reply:
Thanks. I'll change that.